From a8c78ae80ce1ea4934cbf02cbcc8301d82f0d75d Mon Sep 17 00:00:00 2001
From: Stanislav Antonovich <skantonovich@gmail.com>
Date: Wed, 13 May 2020 20:47:02 +0530
Subject: [PATCH 1361/1852] dmaengine: xilinx: axidmatest: In init_srcs remove
 unnecessary buffer increment

commit 3a58fa296af6501a3f906fed6f82d0919847c6da from
https://github.com/Xilinx/linux-xlnx.git

In dmatest_init_srcs() func buffer pointer is initialized at the start
of the loop and accessed throughout using array indexing. Increment of
buffer pointer at the end of the loop is of no use, so remove it.

Signed-off-by: Stanislav Antonovich <skantonovich@gmail.com>
Signed-off-by: Radhey Shyam Pandey <radhey.shyam.pandey@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/dma/xilinx/axidmatest.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/dma/xilinx/axidmatest.c b/drivers/dma/xilinx/axidmatest.c
index c60c214bf4ae..d7658ae90896 100644
--- a/drivers/dma/xilinx/axidmatest.c
+++ b/drivers/dma/xilinx/axidmatest.c
@@ -138,7 +138,6 @@ static void dmatest_init_srcs(u8 **bufs, unsigned int start, unsigned int len)
 				| (~i & PATTERN_COUNT_MASK);
 		for ( ; i < test_buf_size; i++)
 			buf[i] = PATTERN_SRC | (~i & PATTERN_COUNT_MASK);
-		buf++;
 	}
 }
 
-- 
2.31.1

